concordia/static/js/action-app/index.js

```diff
@@ -807,10 +842,18 @@ export class ActionApp {
             alwaysIncludedAssetIDs.add(this.persistentState.get('asset'));
         }
 
-        let currentCampaignId = this.campaignSelect.value;
-        if (currentCampaignId) {
+        let currentCampaignSelectValue = this.campaignSelect.value;
+        let currentCampaignId;
+        let currentTopicId;
+        if (currentCampaignSelectValue) {
+            let campaignOrTopic = this.getCampaignOrTopic();
+
             // The values specified in API responses are integers, not DOM strings:
-            currentCampaignId = parseInt(currentCampaignId, 10);
```

Comment:
We could remove this block by converting `currentCampaignSelectValue` first so there's only one `parseInt` call.

Reply:
I can de-dupe the `parseInt` call but I only want `currentCampaignId` to be defined if it's a campaign that was selected and not a topic.

Reply:
The de-dupe is what I was thinking about. Leaving the variables is a little busy but the ways to refactor that would either take repeated extra string comparisons against `campaignOrTopic`, function calls, etc. and I'd prefer not to tackle something bigger yet.